docs/design/2021-10-22-Enhance-Split-Table-Stability.md

```diff
@@ -0,0 +1,71 @@
+# Proposal: Enhance Split Table Stability and Usability
+
+- Author(s): [hzh](https://github.com/hzh0425)
+- Discussion PR: https://github.com/tikv/pd/issues/4095
+- Tracking Issue: https://github.com/pingcap/tidb/issues/29034
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [Motivation or Background](#motivation-or-background)
+* [Detailed Design](#detailed-design)
+* [Test Design](#test-design)
+* [Follow-up work](#Follow-up work)
+
+## Introduction
+
+Currently, the split table(regions) in tidb is constructed by following 2 steps:
+
+1. Notify tikv to split regions
+2. Notify pd to scatter regions
+
+But this will let the time cost for the whole splitting table become unstable and rather time-consuming.
```

Comment:
The reason why the whole process unstable and time-consuming is due to 2 reasons described in https://github.com/pingcap/tidb/issues/22969. we also need to explain it.